### ISSUE TITLE
Build Instructions Fail

### ISSUE THREAD

**Issue by boyter** [pinpt/ripsrc#17]
```
$ go get -u github.com/pinpt/ripsrc


package github.com/russross/blackfriday/v2: cannot find package "github.com/russross/blackfriday/v2" in any of:
        /usr/local/go/src/github.com/russross/blackfriday/v2 (from $GOROOT)
        /home/bboyter/Go/src/github.com/russross/blackfriday/v2 (from $GOPATH)
```
A simple fix, but it does mean people wanting to try this out cannot build from source straight away.

**Comment by jhaynie** [comment]
Will fix. Thanks for report

**Comment by jhaynie** [comment]
looked into this. when using dep locally it works fine but when using go get it fails. seems to be an interface change in one of our dependencies that we'll need to upgrade (and update our impl) to make this work.

**Comment by boyter** [comment]
Controversial opinion, but perhaps check in dependencies in vendor folder? I have always thought the ability to build from a git pull without another step is a benefit.

**Comment by jhaynie** [comment]
yeah, we used to do that (on other go projects).  we ran into numerous issues. so while it fixes one problem, it opens up other problems too.

**Comment by boyter** [comment]
Oh? Interesting. I have not hit a single issue with it so far. Would love to hear what issues you ran into.

**Comment by jhaynie** [comment]
The biggest ones for us were that the vendor directory becomes a massive set of objects in Git history and if you change / revendor a lot, your git clones become super slow. 

Also, from a commit log standpoint, if you need to make some changes and you have vendor changes along side them -- the commit diff / PR becomes a bit noisy and hard to grok with all these various unrelated changes in your vendor directory that show in the diff.

The biggest benefit is easiest and faster deployment / build since you can just clone and run.  The other nice benefit is that we have had issues (like above) where it can be difficult if a third-party changes and doesn't do semantic versioning well and you have code that doesn't mesh and you get into an unpredictable state.

**Comment by boyter** [comment]
All valid reasons. Thanks for letting me know your thoughts.

**Comment by BrainBuzzer** [comment]
Hi, any updates on this issue?

**Comment by xzandrew** [comment]
@BrainBuzzer If you can, please use dep for installing dependencies, that works without issues. Not supporting `go get` install right now.
